### PR TITLE
Get learning rate per epoch from model history

### PR DESCRIPTION
**Pull request by MahtaFetrat**
The learning rate per epoch is provided by the keras model fit history. Instead of creating the learning rate using numpy arange, we can get it from the history dictionary which seems a neater way to do that.

### CODE REVIEW

**Comment by andres-zartab** [comment]
Thanks for the suggestion, right now we are not merging PR's but we will consider this suggestion for future iterations of the course.
